docs/guides/setup/setup_overview.md

```diff
@@ -0,0 +1,70 @@
+:::
+	
+Getting started with Great Expectations is quick and easy.  Once you have completed your setup, you will have access to all of the features of Great Expectations from a single entry point: Your Data Context.  You will also have your Metadata Stores and Data Docs configured in the manner most suitable for your project's purposes.
+
+## The Setup process
+
+<!-- Brief outline of what the process entails.  -->
+
+Setup entails ensuring your system is prepared to run Great Expectations, installing Great Expectations itself, initializing your deployment, and doing some optional configuration of things like Data Docs and Datastores.  We'll look at each of these things in sequence.
+
+<!-- The following subsections should be repeated as necessary.  They should give a high level map of the things that need to be done or optionally can be done in this process, preferably in the order that they should be addressed (assuming there is one). If the process crosses multiple steps of the Universal Map, use the <SetupHeader> <ConnectHeader> <CreateHeader> and <ValidateHeader> tags to indicate which Universal Map step the subsections fall under. -->
+
+### 1. System Dependencies
+
+The first thing to take care of is making sure your work environment has the utilities you need to install and run Great Expectations.  These include a working Python install (version 3.8 or greater), the ability to pip install Python packages, an internet connection, and a browser so that you can use Jupyter notebooks.  Best practices are to use a virtual environment for your project's workspace.
+
+If you are having trouble with any of these, our documentation on [Supporting Resources](../../reference/supporting_resources.md) will direct you to more information and helpful tutorials.
+
+## 2. Installation
+
+Installing Great Expectations is a simple pip command.  From the terminal, execute:
+
+```markup title="Terminal command:"
+pip install great_expectations
+```
+
+Running this command in an environment configured to accept Python pip install commands will handle the entire installation process for Great Expectations and its dependencies.
+
+## 3. Initialize a Data Context
+
+Your Data Context contains the entirety of your Great Expectations project and provides the entry point for all of the primary methods you will use to configure and interact with Great Expectations.  That's why the first thing you'll do once you've installed Great Expectations will be to initialize your Data Context.  And this is another one-line command.  Simply go to the root folder for your project and execute:
+
+```markdown title="Terminal command:"
+great_expectations --v3-api init
+```
+
+Running this command will initialize your Data Context in the directory that the command is run from.  It will create the folder structure a Data Context requires to organize your project.
+
+## 4. Optional Configurations
```

Comment:
This would be a great section for a diagram to show how the pieces fit together.

Here's a rough draft, with some draft notes. I don't love these---feel like I'm probably another conversation or two away from nailing down the groupings that I could really stand behind.

![Components of a Data Context](https://user-images.githubusercontent.com/1239085/147792073-9c47f790-bf68-40cf-b735-a9935f9c18b6.jpg)

### What are the top-level components within a Great Expectations DataContext?

There are 9 top-level components in a DataContext configuration.

Four of these components fit within the three required steps for setting up Great Expectations.

	* Connect to data: DataSources
	* Create and edit Expectations:
		* ExpectationStore
		* ProfilerStore
	* Validate data : CheckpointStore

Three of the components are intended to be managed internally by Great Expectations.

	* DataDocs
	* ValidationStore
	* EvaluationParameterStore (aka MetricStore)

Two are Advanced:
	
	* plugins
	* config_variables_path (Maybe rename this to SecretsStore?)

Reply:
I'm not sure if we want to insert this as a diagram here, which will make this section seem much longer than the others.  It might be better to have be a reference page that we link to from the Wrapping up section (which we could also have the other Overview pages link to, given that the elements encompassed in this diagram will be relevant across steps).  Something like: "For more information on how Great Expectations elements are organized within a Data Context, see our reference diagram for Great Expectations: Hierarchy of Components" while other Overview pages would have something more like: "For more information on how Profilers and Validators fit into the Great Expectations architecture, see our reference diagram..."

Thoughts?

Reply:
I figure it'll go in one of those two places.

I think it depends on whether you want the high-level "How this thing relates to other things" to live at the "Overview" level or the "Reference" level. FWIW, I'm imagining that each of the 4 steps has a diagram as an important part of its explanation.

IMO, one diagram at the Overview level makes sense, because
1. the Getting Started tutorial is primarily a list of steps and this would let Overview be something more than a list of steps, and
2. in most cases, there'll be another, more intricate diagram that can live in Reference.

I'll be the first to admit that these are pretty soft considerations.

Reply:
I think that makes sense.  I may tweak exactly where in the overview documentation the diagram exists -- I'd like to maintain a certain level of uniformity between documents of the same type, so if there's a place that makes sense to have the 'overview diagram' in all four overview pages, I'd like to standardize that.

Reply:
Per our meeting, I'm going to mark this conversation as resolved and move on with merging this PR.  We'll add illustrations to each of the overview pages in separate PRs so that hashing out the exact message and content of the illustrations doesn't put a hold the initial content updates to our documentation.